var/spack/repos/builtin/packages/povray/package.py

```diff
@@ -147,3 +147,9 @@ def configure_args(self):
         extra_args.append('--without-x')
 
         return extra_args
+
+    def test(self):
```

Comment:
Why not test the installation with the `--benchmark` option and check that the output matches some expectation?

Reply:
Okay.  Good to know it isn't a sufficient test.